maintenance/populateHashField.php

```diff
@@ -290,5 +290,5 @@ public function populate( \Iterator $rows = null ) {
 
```

Comment:
I think you missed the report name here. See [0].

```
[30;43mDeprecation Notice: Class SMW\Maintenance\PopulateHashField located in ./extensions/SemanticMediaWiki/maintenance/populateHashField.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///home/travis/.phpenv/versions/7.1.11/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201[39;49m
[32mGenerated optimized autoload files containing 3399 classes[39m
```

[0] https://api.travis-ci.org/v3/job/665352006/log.txt

Reply:
No idea, but it appears that I completely failed in this effort, meaning I did not change anything at all.

Reply:
> No idea, but it appears that I completely failed in this effort,

Why? You just need to rename the the missing report name here all others seems to be fine after looking at [1] (you need to look for after `composer dump-autoload`).

[0] https://api.travis-ci.org/v3/job/665340570/log.txt (before the change)
[1] https://api.travis-ci.org/v3/job/665352006/log.txt (after the change)

Reply:
Ah, I got stuck at the 5 output of "Generating optimized autoload files" which was discouraging. The 6 one looks nicer. :)

Reply:
Ok, we should be cool now. I will now upgrade my computer. If everything goes well I will be back in one to two hours. Touching wood.

Reply:
> the 5 output of "Generating optimized autoload files" which was

Those belong to the packagist loaded files which doesn't include your PR yet which is why it still generates those messages but when the `dump-autoload` kicks in it includes changes from this PR and the reason why the before after looks different with only `Deprecation Notice: Class SMW\Maintenance\PopulateHashField` being the leftover.

Reply:
Yeah, after a second look I noticed that this must be the case. I already got discouraged after seeing the first lot of composer grump.

PS Now finished to upgrade my computer. It was a two hour walk through disaster but I am functional again though not fully.